Makefile

```diff
@@ -358,11 +360,13 @@ promote-agbot:
 promote-mac-pkg-and-docker: promote-mac-pkg promote-docker promote-agbot
 
 anax-k8s-image:
-	rm -rf $(ANAX_K8S_CONTAINER_DIR)/bin/*
-	cp $(EXECUTABLE) $(ANAX_K8S_CONTAINER_DIR)/bin 
-	cp $(CLI_EXECUTABLE) $(ANAX_K8S_CONTAINER_DIR)/bin
-	@echo "Producing ANAX K8S docker image $(ANAX_K8S_IMAGE_STG)"
+	rm -rf $(ANAX_K8S_CONTAINER_DIR)/anax
```

Comment:
Remove the anax-k8s-push-only and anax-k8s-promote targets because they wont be needed.